src/components/Molecules/SingleMessage/SingleMessage.js

```diff
@@ -71,32 +72,33 @@ const SingleMessage = ({
 
   /* Sets up YT players once all of the ID stuff is taken care of */
   useEffect(() => {
-    if (hasVideo && uniqueID && !isInitialised) {
-      // Switch state to ensure this only runs once per video row
+    if (hasVideo && onPlay && uniqueID && !isInitialised) {
       setIsInitialised(true);
-
+      // Switch state to ensure this only runs once per video row
       // Instantiate a YT Player into our array, using its unique id as the key that PlayButton can access
       allPlayers[uniqueID] = YouTubePlayer(uniqueID, {
         videoId: videoID,
         playerVars: { rel: 0, modestbranding: 1, fs: 0 }
       });
     }
-  }, [hasVideo, isInitialised, uniqueID, videoID]);
+  }, [hasVideo, isInitialised, uniqueID, videoID, onPlay]);
 
   const handlePlay = thisUniqueID => {
+    setonPlay(true);
     // Trigger play and update video state
     const thisVideoID = `${thisUniqueID}__video`;
 
-    allPlayers[thisVideoID].playVideo();
+    setTimeout(() => {
```

Comment:
Hmm.. is there not a nicer way of doing this? There's no guarantee that this'll load in time on a slow connection

Reply:
As per our chat this will load the iframe yt player. Just tested with multi videos and it's working.